src/steth/PriceConverter.sol

```diff
@@ -17,13 +18,17 @@ contract PriceConverter is AccessControl {
     using FixedPointMathLib for uint256;
 
     IwstETH constant wstETH = IwstETH(C.WSTETH);
+    ERC4626 constant sDai = ERC4626(C.SDAI);
 
     event UsdcToEthPriceFeedUpdated(address indexed admin, address newPriceFeed);
     event StEthToEthPriceFeedUpdated(address indexed admin, address newPriceFeed);
 
     // Chainlink price feed (USDC -> ETH)
     AggregatorV3Interface public usdcToEthPriceFeed = AggregatorV3Interface(C.CHAINLINK_USDC_ETH_PRICE_FEED);
 
+    // Chainlink price feed (DAI -> ETH)
+    AggregatorV3Interface public daiToEthPriceFeed = AggregatorV3Interface(C.CHAINLINK_DAI_ETH_PRICE_FEED);
```

Comment:
this is missing a setter function similar to other price feeds

Reply:
Amazing. Thanks for pointing this out. [Added here.](https://github.com/lindy-labs/sandclock-contracts/pull/155/commits/65c6ac6341a0cdf1fb9e265d077825cfed6c08db)